packages/server/migrations/201908131532000-table-global-settings.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  up: async function (queryInterface, Sequelize) {
+    try {
+      await queryInterface.sequelize.query(`BEGIN`);
+
+      await queryInterface.createTable('GlobalSettings', {
+        id: {
+          allowNull: false,
+          primaryKey: true,
+          type: Sequelize.UUID
+        },
+        name: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true,
+        },
+        entity: {
+          type: Sequelize.JSON,
+          allowNull: true,
+        },
+        createdAt: {
+          type: Sequelize.DATE,
+          defaultValue: Sequelize.NOW,
+        },
+        updatedAt: {
+          type: Sequelize.DATE,
+          defaultValue: Sequelize.NOW,
+        },
+        removedAt: {
+          type: Sequelize.DATE,
+          allowNull: true,
+        },
```

Comment:
why do we need a removedAt?

Reply:
One of the good practice is not to remove objects but just mark them as "removed", so for the future system growing or prevent future changes in this table I would like to add this column. But it is not really necessary and I may remove "removedAt" column. Should I remove column since is not clear why the column had been added?

Reply:
Yea. From what I understand, we'll only really have one row, and won't ever delete it, so I don't think this would be necessary.

Reply:
May be there will be a need to store something else instead of not just one file? So in this case this table "GlobalSettings" are standing like a common place of storing data.

Reply:
I'm thinking that we may want to also store settings, like, "defaultAggregatorBotId". However, even here there would just be one row.

If we wanted multiple options of Ken items, then we would make a separate ``Entity`` table or similar.

Reply:
Ok, so I am going to remove "removedAt" column according to these comments.